core/src/gl/vertexLayout.cpp

```diff
@@ -76,6 +76,10 @@ void VertexLayout::enable(const fastmap<std::string, GLuint>& _locations, size_t
 
 }
 
+void VertexLayout::clearCache() {
+    s_enabledAttribs.clear();
```

Comment:
For modifying a static member, it makes more sense to me to use a static function. Then we can just call it once on invalidation, perhaps inside `VboMesh::invalidateAllVbos()`